redfish-core/lib/assembly.hpp

```diff
@@ -106,6 +106,86 @@ inline void
                 std::array<const char*, 0>{});
         }
 
+        // Special handling for LCD and base panel CM.
+        if (sdbusplus::message::object_path(assembly).filename() == "panel1" ||
```

Comment:
And these are only found on everest? Hence on Rainier this ReadyToRemove won't be present ?

Reply:
The name "panel1" and "panel0" are specific to Everest systems and hence these check will fail for Rainier systems.